armi/plugins.py

```diff
@@ -309,10 +309,12 @@ def defineBlockTypes() -> List:
         corresponding ``Component`` type that should activate it. For instance a
         ``HexBlock`` would be created when the largest component is a ``Hexagon``::
 
+            [(Hexagon, HexBlock)]
+
         Returns
         -------
         list
-            [(Hexagon, HexBlock)]
+            ``[(compType, BlockType), ...]``
```

Comment:
I understand this fixed the RST, but is it still correct?

Reply:
According to the docstring above this, it is correct. But I can switch to what was originally there if I'm mistaken.